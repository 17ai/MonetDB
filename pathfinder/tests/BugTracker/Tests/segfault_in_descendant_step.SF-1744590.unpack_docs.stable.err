stderr of test 'segfault_in_descendant_step.SF-1744590.unpack_docs` in directory 'tests/BugTracker` itself:


# 16:23:55 >  
# 16:23:55 >  Mtimeout -timeout 60 /usr/bin/python segfault_in_descendant_step.SF-1744590.unpack_docs.py segfault_in_descendant_step.SF-1744590.unpack_docs 
# 16:23:55 >  


# 16:23:55 >  
# 16:23:55 >  Done.
# 16:23:55 >  

