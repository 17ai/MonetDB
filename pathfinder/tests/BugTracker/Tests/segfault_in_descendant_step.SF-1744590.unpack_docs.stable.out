stdout of test 'segfault_in_descendant_step.SF-1744590.unpack_docs` in directory 'tests/BugTracker` itself:


# 16:23:55 >  
# 16:23:55 >  Mtimeout -timeout 60 /usr/bin/python segfault_in_descendant_step.SF-1744590.unpack_docs.py segfault_in_descendant_step.SF-1744590.unpack_docs 
# 16:23:55 >  

Archive:  segfault_in_descendant_step.SF-1744590.unpack_docs.zip
  inflating: segfault_in_descendant_step.SF-1744590.xml

# 16:23:55 >  
# 16:23:55 >  Done.
# 16:23:55 >  

